app/assets/stylesheets/banner.scss

```diff
@@ -0,0 +1,11 @@
+.banner {
+  margin-bottom: 2rem;
+
+  &__container {
```

Comment:
Instead of adding an extra container, do we get the same results if we apply these styles to the `a` tag inside the `.banner`? If so, I'd rather go for it; we already have too many `div` tags in our views :sweat_smile:.